src/js/media-time-range.js

```diff
@@ -535,7 +540,7 @@ class MediaTimeRange extends MediaChromeRange {
 
     // Get mouse position percent
     const rangeRect = this.range.getBoundingClientRect();
-    let mouseRatio = (evt.clientX - rangeRect.left - this.thumbWidth / 2) / (rangeRect.width - this.thumbWidth);
+    let mouseRatio = (evt.clientX - rangeRect.left) / rangeRect.width;
```

Comment:
praise: I originally had a question here since this was an annoying edge case we had to account for... but instead, with your new bespoke UI, it's another improvement! 🎉